base_global_discount/models/res_partner.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    customer_global_discount_ids = fields.Many2many(
```

Comment:
For a good visualization, add a `name_get` on `global.discount` with "Name (amount%)" or "Name (-amount)" depending on the type of discount.

Reply:
done